packages/configure-splash-screen/README.md

```diff
@@ -32,32 +33,30 @@ You can use it to configure your native iOS and Android project according to you
 
-- `-r, --resize-mode [resizeMode]` - ResizeMode to be used for native splash screen image. Available values: "contain" | "cover" | "native" (only available for android platform)) (default: "contain"). See [resize modes](https://github.com/expo/expo/tree/master/packages/expo-splash-screen#built-in-splash-screen-image-resize-modes) for more information.
-- `-p, --platform [platform]` - Selected platform to configure. Available values: "android" | "ios" | "all". (default: "all")
-- `--statusbar-style [statusBarStyle]` - Customizes the color of the StatusBar icons. Available values: "default" | "light-content" | "dark-content". (default: "default")
-- `--dark-mode-statusbar-style [darkModeStatusBarStyle]` - (only for Android platform) The very same as 'statusbar-style' option, but applied only in dark mode. Available only if 'statusbar-style' is provided.
-- `--statusbar-hidden` - Hides the StatusBar.
-- `--statusbar-background-color [statusBarBackgroundColor]` - (only for Android platform) Customizes the background color of the StatusBar. Valid css-formatted color (see backgroundColor supported formats).
-- `--dark-mode-statusbar-background-color [darkModeStatusBarBackgroundColor]` - (only for Android platform) The very same as 'statusbar-background-color' option, but applied only in dark mode. Available only if `statusbar-style` is provided.
-- `--statusbar-translucent` - (only for Android platform) Makes the StatusBar translucent (enables drawing under the StatusBar area).
+- `-p, --platform <platform>` - Selected platform to configure. Available values: "android" | "ios" | "all" (default: "all").
+- `-b, --background-color <color>` - (required) Valid css-formatted color (hex (#RRGGBB[AA]), rgb[a], hsl[a], named color (https://drafts.csswg.org/css-color/#named-colors)) that would be used as the background color for native splash screen view.
+- `-i, --image-path <path>` - Path to valid .png image that will be displayed in native splash screen.
+- `-r, --image-resize-mode <resizeMode>` - ResizeMode to be used for native splash screen image. Available only if 'image-path' is provided as well. Available values: "contain" | "cover" | "native" (only available for android platform)) (default: "contain"). See [resize modes](https://github.com/expo/expo/tree/master/packages/expo-splash-screen#built-in-splash-screen-image-resize-modes) for more information.
+- `--dark-mode-background-color <color>` - Color (see 'background-color' supported formats) that would be used as the background color for native splash screen in dark mode. Providing this option enables other dark-mode related options.
+- `--dark-mode-image-path <path>` - Path to valid .png image that will be displayed in native splash screen in dark mode only. Available only if 'dark-mode-background-color' is provided as well.
+- `--status-bar-style <style>` - Customizes the color of the StatusBar icons. Available values: "default" | "light-content" | "dark-content" (default: "default").
```

Comment:
```suggestion
- `--status-bar-style <style>` - Customizes the color of the status bar icons. Available values: "default" | "light-content" | "dark-content" (default: "default").
```

Reply:
not necessary for this PR but maybe we should change "light-content" and "dark-content" to just "light" and "dark" like in https://docs.expo.io/versions/latest/sdk/status-bar/#statusbarstyle